tasks/configuration.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Create configuration directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: "{{ icingaweb2_unix_group }}"
+    mode: 0755
+  with_items:
+    - /etc/icingaweb2/enabledModules
+    - /etc/icingaweb2/modules/monitoring
+
+- name: Create configuration files
+  template:
+    src: "{{ item }}.ini.j2"
+    dest: "/etc/icingaweb2/{{ item }}.ini"
+    owner: root
+    group: "{{ icingaweb2_unix_group }}"
+    mode: 0640
+  with_items:
+    - authentication
+    - config
+    - groups
+    - resources
+    - roles
+
+- name: Enable modules
+  file:
+    src: "{{ icingaweb2_module_path.split(':') | first }}/{{ item }}"
+    dest: "/etc/icingaweb2/enabledModules/{{ item }}"
+    state: link
+    owner: root
+    group: "{{ icingaweb2_unix_group }}"
+    force: true # Because otherwise, Ansible won't symlink directories
+  with_items:
+    - "{{ icingaweb2_modules }}"
+    - unicorn
```

Comment:
Why not add unicorn to `icingaweb2_modules`, then you don't have to check it specifically two tasks down. Just a suggestion.

Reply:
No, because one can override `icingaweb2_modules`

Reply:
I meant amending it using `set_fact`